common.mk

```diff
@@ -381,6 +381,10 @@ QEMU_EXTRA_ARGS +=\
 	  -device virtio-9p-device,fsdev=fsdev1,mount_tag=secure
 endif
 endif
+ifeq ($(QEMU_GDB_ENABLE),y)
+# disable kaslr of the kernel to get symbolic debugging with gdb
+QEMU_GDB_BOOTARGS ?= nokaslr
+endif
```

Comment:
Why introduce two config variables? It seems `QEMU_GDB_BOOTARGS` is enough. And it had better be named `QEMU_KERNEL_BOOTARGS` I think. And I don't think it belongs in `common.mk`, simply appending it to the kernel command line in `qemu_v8.mk` should be OK.

Reply:
My reasoning for putting it in common.mk was mainly because a bunch of other qemu config stuff was there. But I just noticed QEMU_SMP ?= 2 is also set in qemu_v8.mk.

In v2 I have done as you suggest and renamed to QEMU_KERNEL_BOOTARGS, and moved it to qemu_v8.mk. I've left the default as nothing (as maybe turning off kaslr by default isn't what we want). But it can be passed through in the following way from cmd line if you want to debug the kernel with gdb (and same mechanism could be used for other useful debug kernel bootargs during development)

make QEMU_KERNEL_BOOTARGS=nokaslr run-only